pom.xml

```diff
@@ -459,10 +459,35 @@
 
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>default</id>
+                        <phase>verify</phase>
+                        <goals>
+                            <goal>enforce</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <rules>
+                        <requireUpperBoundDeps />
+                        <banDuplicatePomDependencyVersions />
+                        <requireMavenVersion>
+                            <version>3.5.0</version>
+                        </requireMavenVersion>
+                        <requireJavaVersion>
+                            <version>${project.build.targetJdk}</version>
```

Comment:
Is this required? Airbase should cover the enforcer

Reply:
Yes, I verified all the rules are already present in airbase, I have removed the plugin from this pom.